### ISSUE TITLE
When no onChange handler, component throws error

### ISSUE THREAD

**Issue by miuirussia** [scniro/react-codemirror2#68]
```
TypeError: _this.props.onChange is not a function. (In '_this.props.onChange(_this.editor, data, _this.editor.getValue())', '_this.props.onChange' is undefined)
```

Probably due to lack of verification:
https://github.com/scniro/react-codemirror2/blob/a78ade3804bb922d2c407051218d19ec93b7650c/src/index.tsx#L680-L687

**Comment by scniro** [comment]
@miuirussia yea this looks like an issue. I can add a check for it. You should just add an `onChange` for now...

**Comment by scniro** [comment]
@miuirussia fixed with [4.2.1](https://github.com/scniro/react-codemirror2/releases/tag/4.2.1)
